objects/_objects.ratio.scss

```diff
@@ -83,3 +83,26 @@ $inuit-ratios: (
 }
 
 /* stylelint-enable */
+
+
+
+
+
+/* Contain modifiers.
+   ========================================================================== */
+
+/**
+ * Only works with image content.
+ */
+
+.o-ratio--contain {
```

Comment:
As it only works for image content, we should probably match the name of the class as well.

Reply:
Second that. Something like `.o-ratio--img-contain` would be more concise.

Reply:
@anselmh @csshugs you're right, I'll update that today.